presentation/src/main/kotlin/com/jshvarts/conductormvp/viewext/Extensions.kt

```diff
@@ -0,0 +1,11 @@
+package com.jshvarts.conductormvp.viewext
+
+import android.support.v7.widget.RecyclerView
+
+fun RecyclerView.initRecyclerView(layoutManager: RecyclerView.LayoutManager,
+                                  adapter: RecyclerView.Adapter<out RecyclerView.ViewHolder>,
+                                  hasFixedSize: Boolean = true) {
+    this.layoutManager = layoutManager
+    this.adapter = adapter
+    setHasFixedSize(hasFixedSize)
```

Comment:
property access? `this.hasFixedSize = hasFixedSize`

Reply:
it's only exposed via getter and setter